tutor/templates/local/docker-compose.yml

```diff
@@ -44,7 +44,12 @@ services:
       --character-set-server=utf8mb4
       --collation-server=utf8mb4_unicode_ci
       --binlog-expire-logs-seconds=259200
+      # We only require this option for MySQL 8.4.0 and above
+      # Breaks MySQL for previous versions as this option does not exist on versions earlier than 8.4.0
+      {% set mysql_version = DOCKER_IMAGE_MYSQL.split(':')[-1] -%}
```

Comment:
nit: please do the same in k8s file as well.

Reply:
Done.